release/release.go

```diff
@@ -27,7 +49,11 @@ func main() {
 	switch cmd {
 	case "build-archive":
 		buildArchive()
-	case "build-installer":
+	case "build-msi":
+		buildMSI()
+	case "build-rpm":
+		log.Fatal("not implemented")
+	case "build-deb":
```

Comment:
we've already got a `build-linux` below for linux packages; I don't think we need these rpm and deb specific cases